components/templates/src/builtins/atom.xml

```diff
@@ -48,6 +48,9 @@
         {% else %}
         <content type="html" xml:base="{{ page.permalink | escape_xml | safe }}">{{ page.content }}</content>
         {% endif %}
+
+        {%- block atom_entry_extra %}
+        {% endblock atom_entry_extra -%}
```

Comment:
I still don't understand why, but these lines cause the tests to fail:

```
     Running tests/invalid.rs (/home/h/projects/zola/target/debug/deps/invalid-9ae7bc0e52bdd68b)

running 1 test
test errors_on_index_md_page_in_section ... FAILED

failures:

---- errors_on_index_md_page_in_section stdout ----
thread 'errors_on_index_md_page_in_section' panicked at 'called `Result::unwrap()` on an `Err` value: Error { kind: Msg("Failed to parse '__zola_builtins/atom.xml'"), source: Some(Error { kind: Msg("  --> 52:9\n   |\n52 |         {% block atom_entry_extra %}\n   |         ^---\n   |\n   = unexpected tag; expected an `else` tag, an endfor tag (`{% endfor %}`), or some content"), source: None }) }', components/templates/src/lib.rs:28:6
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

The message is:

```bash
$ echo -e "  --> 52:9\n   |\n52 |         {%- block atom_entry_extra %}\n   |         ^---\n   |\n   = unexpected tag; expected an `else` tag, an endfor tag (`{% endfor %}`), or some content"
  --> 52:9
   |
52 |         {%- block atom_entry_extra %}
   |         ^---
   |
   = unexpected tag; expected an `else` tag, an endfor tag (`{% endfor %}`), or some content
```

We can't add a block to a builtin template? I am not sure what I am missing here.

Reply:
It's because the block is in a for loop, which is not allowed in Tera. Maybe the extend approach doesn't work then.